malleefowl/custom_workflow.py

```diff
@@ -0,0 +1,266 @@
+from owslib.wps import WebProcessingService
+from owslib.wps import ComplexDataInput
+
+from dispel4py.workflow_graph import WorkflowGraph
+from dispel4py import simple_process
+from dispel4py.core import GenericPE
+
+from malleefowl.workflow import run as run_basic_workflow
+
+import logging
+logger = logging.getLogger("PYWPS")
```

Comment:
Might be better to instanciate the logger in a function otherwise it will be done at import time, which might create the logger before the logger was configured...